utils/lru_test.go

```diff
@@ -20,49 +21,42 @@ func TestLRU(t *testing.T) {
 	for i := 0; i < 256; i++ {
 		l.Add(i, i)
 	}
-	if l.Len() != 128 {
-		t.Fatalf("bad len: %v", l.Len())
-	}
+	require.Equalf(t, 128, l.Len(), "bad len: %v", l.Len())
 
 	for i, k := range l.Keys() {
-		if v, ok := l.Get(k); !ok || v != k || v != i+128 {
-			t.Fatalf("bad key: %v", k)
-		}
+		v, ok := l.Get(k)
+		require.Equalf(t, true, ok, "bad key: %v", k)
+		require.Equalf(t, v, k, "bad key: %v", k)
+		require.Equalf(t, i+128, v, "bad key: %v", k)
 	}
 	for i := 0; i < 128; i++ {
 		_, ok := l.Get(i)
-		if ok {
-			t.Fatalf("should be evicted")
-		}
+		require.NotEqual(t, true, ok, "should be evicted")
 	}
 	for i := 128; i < 256; i++ {
 		_, ok := l.Get(i)
-		if !ok {
-			t.Fatalf("should not be evicted")
-		}
+		require.NotEqualf(t, false, ok, "should not be evicted")
 	}
 	for i := 128; i < 192; i++ {
 		l.Remove(i)
 		_, ok := l.Get(i)
-		if ok {
-			t.Fatalf("should be deleted")
-		}
+		require.NotEqual(t, true, ok, "should be deleted")
 	}
 
 	l.Get(192) // expect 192 to be last key in l.Keys()
 
 	for i, k := range l.Keys() {
 		if (i < 63 && k != i+193) || (i == 63 && k != 192) {
```

Comment:
We should include these conditions into booleans and use `require.True` to check for these to avoid a this `require.Fail`